src/coreComponents/mesh/FaceElementSubRegion.cpp

```diff
@@ -1048,4 +1048,104 @@ std::set< std::set< globalIndex > > FaceElementSubRegion::getCollocatedNodes() c
   return result;
 }
 
+void FaceElementSubRegion::flipFaceMap( FaceManager & faceManager,
+                                        ElementRegionManager const & elemManager )
+{
+  ArrayOfArraysView< localIndex > const & elems2dToFaces = faceList().toView();
+  arrayView2d< localIndex const > const & faceToElementRegionIndex    = faceManager.elementRegionList();
+  arrayView2d< localIndex const > const & faceToElementSubRegionIndex = faceManager.elementSubRegionList();
+  arrayView2d< localIndex const > const & faceToElementIndex          = faceManager.elementList();
+
+  ElementRegionManager::ElementViewAccessor< arrayView1d< globalIndex const > > const cellElemGlobalIndex =
+    elemManager.constructArrayViewAccessor< globalIndex, 1 >( ObjectManagerBase::viewKeyStruct::localToGlobalMapString() );
+
+  forAll< parallelHostPolicy >( this->size(), [=]( localIndex const kfe )
+  {
+    if( elems2dToFaces.sizeOfArray( kfe ) != 2 )
+    {
+      return;
+    }
+
+    localIndex const f0 = elems2dToFaces[kfe][0];
+    localIndex const f1 = elems2dToFaces[kfe][1];
```

Comment:
```suggestion
    localIndex & f0 = elems2dToFaces[kfe][0];
    localIndex & f1 = elems2dToFaces[kfe][1];
```
then you can `std::swap( f0, f1 )` down below.
At your convenience.